templates/password_changed.txt

```diff
@@ -0,0 +1,5 @@
+{{t "Password Change Successfully" }}
```

Comment:
"Change" or "Changed" (past tense)?
If it's "Change" we may need more words.
If it's "Changed" then it's probably fine and matches the "successfully changed" text on line 2 below.